internal/services/compute/windows_virtual_machine_scale_set_network_resource_test.go

```diff
@@ -33,6 +37,8 @@ func TestAccWindowsVirtualMachineScaleSet_networkAcceleratedNetworkingUpdated(t
 			Config: r.networkAcceleratedNetworking(data, false),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
+				// Accelerated Networking can only be changed when VM is not running
+				data.CheckWithClientForResource(r.deallocateVirtualMachineScaleSet(), "azurerm_windows_virtual_machine_scale_set.test"),
```

Comment:
this is only fixing the test not the reason for its failure, users will still see the issue? so we should detect this change and then do this in the update function?

Reply:
Make sense. I took another look at the document, and found the tests are using `Standard_F4` that doesn't support accelerated networking, thus the VM has to be deallocated. After changing it to the allowed size `Standard_D2s_v3` it doesn't require the VMto be stopped. I've also added the same missing update test to orchestrated vmss
Ref:
[Standard_F2](https://learn.microsoft.com/azure/virtual-machines/sizes-previous-gen#f-series)[Standard_D2s_v3](https://learn.microsoft.com/azure/virtual-machines/dv3-dsv3-series#dsv3-series)